src/libraries/Microsoft.Win32.SystemEvents/src/Microsoft/Win32/SystemEvents.cs

```diff
@@ -1114,7 +1114,6 @@ private static void Shutdown()
             }
         }
 
-        [PrePrepareMethod]
```

Comment:
_Microsoft.Win32.SystemEvents.csproj_ includes netstandard + netframework as one of its targets. May need to keep this attribute.

Reply:
The file is only consumed there when `$(TargetsWindows)` is true.  @jkotas, can that be true when targeting netstandard or netframework?

Reply:
I just found references to `net461-Windows_NT`, so I believe the answer here is yes.